dns_compare

```diff
@@ -62,6 +62,8 @@ if __name__ == '__main__':
                         help="compare NS records (default: false)")
     parser.add_option("-t","--ttl", dest="compare_ttl", action="store_false", default=True,
                         help="compare TTL values (default: true)")
+    parser.add_option("-c","--cname", dest="lookup_cnames", action="store_true", default=False,
+                        help="lookup cname-values that dosn't match (default: false)")
```

Comment:
doesn't is spelled wrong but a better grammatical fit would be "lookup cname-values that do not match"